spec/lib/oauth/token_request_spec.rb

```diff
@@ -1,6 +1,12 @@
 require 'spec_helper_integration'
 
 module Doorkeeper::OAuth
+  describe ClientCredentialsRequest do
+    subject do
+      # TODO: Do we need something here too?
```

Comment:
I don't think we need to add this indeed, would you remove this new block please?